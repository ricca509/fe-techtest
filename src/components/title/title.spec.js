import Title from "./";

describe("Title", () => {
  it("should render the page title", () => {
    expect(Title()).toMatchSnapshot();
  });
});
