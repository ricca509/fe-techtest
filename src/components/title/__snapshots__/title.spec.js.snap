// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render the page title 1`] = `
"
    <header class=\\"header\\">
      <h1>Let's try multiplications!</h1>
      <h2>Select a number to see all its multiples, click again to make them disappear</h2>
    </header>
  "
`;
